### PR TITLE
Helm chart info in service dashboards cds 73310

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
# Harness Developer Pull Request

Jira ticket: [CDS-73310 Service Dashboard to display Helm Chart version as part of Instance Information](https://harness.atlassian.net/browse/CDS-73310)

Preview links: 
- CD release notes [Latest: Harness version 80711](https://6511e33b55be38074d8eede7--harness-developer.netlify.app/release-notes/continuous-delivery#latest-harness-version-80711)
- Early access [Continuous Delivery, version 80711](https://6511e33b55be38074d8eede7--harness-developer.netlify.app/release-notes/early-access#continuous-delivery-version-80711) 
- Monitor deployments and services in CD dashboards [Chart Versions tab](https://6511e33b55be38074d8eede7--harness-developer.netlify.app/docs/continuous-delivery/monitor-deployments/monitor-cd-deployments/#chart-versions-tab)

Jira ticket: [Service Dashboard to display Helm Chart version as part of Instance Information](https://harness.atlassian.net/browse/CDS-73310)

### CODE REVIEW

File: docs/continuous-delivery/monitor-deployments/monitor-cd-deployments.md [hunk 1]
```diff
@@ -156,6 +156,17 @@ To use custom sequencing, do the following:
   
   You can disable custom sequencing by toggling it off.
 
+### Chart Versions tab
+
+ The **Chart Versions** tab shows tiles for each Helm chart used to deploy the service (and a tile for any non-Helm deployments). Each tile shows the instances/artifact, environment, and latest deployment time. 
+
+<docimage path={require('/docs/continuous-delivery/monitor-deployments/static/cds-77310-charts-tab-callouts.png')} width="100%" height="100%" title="Click to view full size image" />
```
Comment (RishabhGupta34) [inline docs/continuous-delivery/monitor-deployments/monitor-cd-deployments.md:163]: In the chart version tab -> Each card has heading for chart version, but the below entries are for env., in the image we are saying `What artifacts were deployed for each version`, can we fix that?

File: release-notes/continuous-delivery.md [hunk 2]
```diff
@@ -42,7 +42,30 @@ This release does not have new features.
...
+  - The **Chart Versions** tab shows tiles for each Helm chart used to deploy the service (and a tile for any non-Helm deployments). Each tile shows the instances/artifact, environment, and latest time for each deployment. 
+
+    <docimage path={require('./static/cds-77310-chart-tab-panels.png')} width="75%" height="75%" title="Click to view full size image" />   
+
+  - The Pipeline Executions table for the service now shows the Helm chart version in the drilldown information for that execution. 
+
+  
+  :::note
+
+  This information appears only for services that were deployed using Helm charts.
```
Comment (RishabhGupta34) [inline release-notes/continuous-delivery.md:65]: can we add this note to the other page also?

File: release-notes/early-access.md [hunk 3]
```diff
@@ -17,7 +17,28 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
 
 ### Continuous Delivery, version 80711
 
-– Added support for Post Prod Rollback for Native Helm deployment types. For these Services, a Rollback to the previous version can be triggered from the Services Dashboard. For more information, go to [Rollback Deployments](https://developer.harness.io/docs/continuous-delivery/manage-deployments/rollback-deployments). This feature is currently behind the Feature Flag `POST_PROD_ROLLBACK`. Please contact Harness Support to enable. (CDS-67121)
+- Added support for Post Prod Rollback for Native Helm deployment types. For these Services, a Rollback to the previous version can be triggered from the Services Dashboard. For more information, go to [Rollback Deployments](https://developer.harness.io/docs/continuous-delivery/manage-deployments/rollback-deployments). This feature is currently behind the Feature Flag `POST_PROD_ROLLBACK`. Please contact Harness Support to enable. (CDS-67121)
+
+- The Services Dashboard includes new support for Helm Chart deployments. (CDS-73310)
```
Comment (RishabhGupta34) [inline release-notes/early-access.md:22]: This service dashboard itself is behind a FF, should we add that also in release notes?
`CDC_SERVICE_DASHBOARD_REVAMP_NG`

File: docs/continuous-delivery/monitor-deployments/monitor-cd-deployments.md (hunk 1 above)
Comment (douglas-j-bothwell) [reply docs/continuous-delivery/monitor-deployments/monitor-cd-deployments.md:163]: Updated callout to say "Where were these artifacts deployed?"

File: release-notes/continuous-delivery.md [hunk 4]
```diff
@@ -42,7 +42,31 @@ This release does not have new features.
 
 ### Early access features
 
-– Added support for Post Prod Rollback for Native Helm deployment types. For these Services, a Rollback to the previous version can be triggered from the Services Dashboard. For more information, go to [Rollback Deployments](https://developer.harness.io/docs/continuous-delivery/manage-deployments/rollback-deployments). This feature is currently behind the Feature Flag `POST_PROD_ROLLBACK`. Please contact Harness Support to enable. (CDS-78243)
+- Added support for Post Prod Rollback for Native Helm deployment types. For these Services, a Rollback to the previous version can be triggered from the Services Dashboard. For more information, go to [Rollback Deployments](https://developer.harness.io/docs/continuous-delivery/manage-deployments/rollback-deployments). This feature is currently behind the Feature Flag `POST_PROD_ROLLBACK`. Please contact Harness Support to enable. (CDS-78243)
```
Comment (brian-f-harness) [inline release-notes/continuous-delivery.md:45]:
```suggestion
- Added support for Post Prod Rollback for Native Helm deployment types. For these Services, a Rollback to the previous version can be triggered from the Services Dashboard. For more information, go to [Rollback Deployments](/docs/continuous-delivery/manage-deployments/rollback-deployments). This feature is behind the Feature Flag `POST_PROD_ROLLBACK`. Contact [Harness Support](mailto:support@harness.io) to enable the feature. (CDS-78243)
```

File: release-notes/continuous-delivery.md [hunk 5]
```diff
@@ -42,7 +42,31 @@ This release does not have new features.
...
+- The Services Dashboard includes new support for Helm Chart deployments. (CDS-73310)
+
+  :::note Important Notes
+  - This information appears only for services that were deployed using Helm charts. 
+  - Currently, this feature is behind the Feature Flag `CDC_SERVICE_DASHBOARD_REVAMP_NG`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
+  :::
+
+  This dashboard includes the following enhancements:
+
+  - Tiles in the service overview (**Environments** tab) now show the Helm Chart versions (if any) deployed in each environment.
```
Comment (brian-f-harness) [inline release-notes/continuous-delivery.md:56]:
```suggestion
  - Tiles in the service overview (**Environments** tab) now show the Helm chart versions (if any) deployed in each environment.
```

File: release-notes/early-access.md [hunk 6]
```diff
@@ -17,7 +17,30 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
 
 ### Continuous Delivery, version 80711
 
-– Added support for Post Prod Rollback for Native Helm deployment types. For these Services, a Rollback to the previous version can be triggered from the Services Dashboard. For more information, go to [Rollback Deployments](https://developer.harness.io/docs/continuous-delivery/manage-deployments/rollback-deployments). This feature is currently behind the Feature Flag `POST_PROD_ROLLBACK`. Please contact Harness Support to enable. (CDS-67121)
+- Added support for Post Prod Rollback for Native Helm deployment types. For these Services, a Rollback to the previous version can be triggered from the Services Dashboard. For more information, go to [Rollback Deployments](https://developer.harness.io/docs/continuous-delivery/manage-deployments/rollback-deployments). This feature is currently behind the Feature Flag `POST_PROD_ROLLBACK`. Please contact Harness Support to enable. (CDS-67121)
```
Comment (brian-f-harness) [inline release-notes/early-access.md:20]:
```suggestion
- Added support for Post Prod Rollback for Native Helm deployment types. For these Services, a Rollback to the previous version can be triggered from the Services Dashboard. For more information, go to [Rollback Deployments](https://developer.harness.io/docs/continuous-delivery/manage-deployments/rollback-deployments). This feature is behind the Feature Flag `POST_PROD_ROLLBACK`. Contact [Harness Support](mailto:support@harness.io) to enable the feature. (CDS-67121)
```

File: release-notes/early-access.md [hunk 7]
```diff
@@ -17,7 +17,30 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+  
+  - This information appears only for services that were deployed using Helm charts. 
+  
+  - Currently, this feature is behind the Feature Flag `CDC_SERVICE_DASHBOARD_REVAMP_NG`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
+  
+  :::
+
+  This dashboard includes the following enhancements:
+
+  - Tiles in the service overview (**Environments** tab) now show the Helm Chart versions (if any) deployed in each environment.
```
Comment (brian-f-harness) [inline release-notes/early-access.md:34]:
```suggestion
  - Tiles in the service overview (**Environments** tab) now show the Helm chart versions (if any) deployed in each environment.
```

**Review by brian-f-harness — APPROVED** [review]
LGTM. Left a couple of minor suggestions.

**Comment by douglas-j-bothwell** [comment]
@RishabhGupta34 Your edits were pretty straightforward so I implemented them and am merging this PR. 

cc @PrasadRavella

**Pull request merged**